website/build/granary/index.html

```diff
@@ -1,7 +1,11 @@
+</span></div></h2><div><span><p>Configure your AWS resources once, then let Granary keep track of them. Granary associates models with specific execution environments so you don't have to remember what runs where. Coming soon:</p>
+<ul>
+<li>AWS Lambda executor</li>
+<li>HTTP executor</li>
+</ul>
+</span></div></div><div class="blockImage"><img src="/granary/img/undraw_feeling_proud_qne1.svg"/></div></div></div></div></div><div class="container paddingBottom paddingTop" id="try"><div class="wrapper"><div class="gridBlock"><div class="blockElement imageAlignSide imageAlignLeft twoByGridBlock"><div class="blockImage"><img src="/granary/img/undraw_predictive_analytics_kf9n.svg"/></div><div class="blockContent"><h2><div><span><p>Keep track of your predictions and their results</p>
+</span></div></h2><div><span><p>Associating data with the model run that produced it can be hard. With Granary tracking the arguments, outputs, and infrastructure for your tasks, you won't have to guess which configuration produced which results.</p>
+</span></div></div></div></div></div></div><div class="container darkBackground paddingBottom paddingTop"><div class="wrapper"><div class="gridBlock"><div class="blockElement imageAlignSide imageAlignRight twoByGridBlock"><div class="blockContent"><h2><div><span><p>Stop kicking off doomed models</p>
 </span></div></h2><div><span><p>Granary puts some guardrails in front of your interaction with Batch so you don't fall in to common mistakes, like mis-formatted arguments,missing arguments, and bad string values for important AWS resources.</p>
```

Comment:
```suggestion
</span></div></h2><div><span><p>Granary puts some guardrails in front of your interaction with Batch so you don't fall in to common mistakes, like mis-formatted arguments, missing arguments, and bad string values for important AWS resources.</p>
```

Just noticed this space is missing